test/cmdlineTests/standard_urls_existing_and_missing/output.json

```
@@ -0,0 +1,18 @@
+{
+    "errors":
+    [
+        {
+            "component": "general",
+            "formattedMessage": "Cannot import url (\"standard_urls_existing_and_missing/non-existent-contract-1.sol\"): File not found. Searched the following locations: \"\".",
+            "message": "Cannot import url (\"standard_urls_existing_and_missing/non-existent-contract-1.sol\"): File not found. Searched the following locations: \"\".",
+            "severity": "warning",
+            "type": "Warning"
+        }],
```

```suggestion
        }
    ],
```
It seems there should be a `linebreak` here in order to fix the failing test.

---

Although, after running the test locally, it does produce the output without the  `linebreak`.

---

I wonder why `t_ubu_cli` did not fail because of this. Weird.

---

@matheusaaguiar  indeed, the local run of the tests does produce the output without the new line. 

I suspect, for now, I can add the line break on my own and let the tests run on the CI. But we might need to investigate as to why this is happening ?

---

So you mean that running with `--update` option updates the output in such a way? If so then yeah, might be a good idea to look into the `cmdlineTests.sh` script.

---

Should we do that in another issue @cameel , @matheusaaguiar  ? 
For now I've made the changes to get the test cases passed.

---

I would not create an issue out of it. Just play with the script a bit and see if you can figure out why it's doing this (i.e. is this done by the compiler? does it sometimes return one form and sometimes the other?). But don't go too deep. It's not **that** important and not something that will stop this PR. We're done here :)